client/src/components/Common/selectCurrency/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Select } from 'antd';
+import PropTypes from 'prop-types';
+import flagOptions from './currencies';
+
+import './style.css';
+import 'antd/dist/antd.css';
```

Comment:
your style should be under antd style, cause your style should override it's style. 
```suggestion
import 'antd/dist/antd.css';
import './style.css';
```